### ISSUE TITLE
Open new ORA files with Pixelitor without creating a new instance of the application

### ISSUE THREAD

**Issue by Just-Another-Gnumer** [lbalazscs/Pixelitor#163]
First of all, it appears this may be an issue with Java apps as a whole and not Pixelitor in particular.  My work around has been dragging and dropping images into the app but I like to know for sure if there is not a way to solve this.

I can run the bash command

`java -jar /home/USERNAME/Applications/Pixelitor-4.2.4.jar /home/USERNAME/Pictures/Pixelitor_Workstation/peppermint_swirly.ora /home/USERNAME/Pictures/Pixelitor_Workstation/pink_swirl_depth.ora`

and I will successfully open two images in the same Pixelitor instance, but as far as I know it is not possible to open a new file without drag and drop or the file picker.  If I attempt to do this again it will open a new Pixelitor instance.

Do you know of a resolve for this?

**Comment by lbalazscs** [comment]
No, currently the only other way to open a file is the recent files menu, but you probably already considered that. A less obvious feature is that you can drag-and-drop folders and not just individual files, and then Pixelitor will try to open all files with supported extensions.

If you are an advanced Linux wizard, you might be able to write a script that uses xdotool to simulate a drag-and-drop, something like 

https://unix.stackexchange.com/questions/216507/drag-and-drop-without-the-drag

(I didn't try it)
